cmd/milvus/mck.go

```diff
@@ -0,0 +1,692 @@
+		if loadReq.LoadMeta != nil {
+			partitionIDs = append(partitionIDs, loadReq.LoadMeta.PartitionIDs...)
+		}
+		pids, sids := c.extractVchannelInfo(taskID, loadReq.Infos)
+		partitionIDs = append(partitionIDs, pids...)
+		segmentIDs = append(segmentIDs, sids...)
+		log.Info("WatchDeltaChannels", zap.String("detail", fmt.Sprintf("+%v", loadReq)))
+		return "WatchDeltaChannels", removeRepeatElement(partitionIDs), removeRepeatElement(segmentIDs), nil
+	case commonpb.MsgType_WatchQueryChannels:
+		loadReq := querypb.AddQueryChannelRequest{}
+		err = proto.Unmarshal([]byte(t), &loadReq)
+		if err != nil {
+			return errReturn(taskID, "WatchDeltaChannelsRequest", err)
+		}
+		pids, sids := c.extractQuerySegmentInfos(taskID, loadReq.GlobalSealedSegments)
+		log.Info("WatchQueryChannels", zap.String("detail", fmt.Sprintf("+%v", loadReq)))
+		return "WatchQueryChannels", pids, sids, nil
+	case commonpb.MsgType_LoadBalanceSegments:
+		loadReq := querypb.LoadBalanceRequest{}
+		err = proto.Unmarshal([]byte(t), &loadReq)
+		if err != nil {
+			return errReturn(taskID, "LoadBalanceRequest", err)
+		}
+		log.Info("LoadBalanceSegments", zap.String("detail", fmt.Sprintf("+%v", loadReq)))
+		return "LoadBalanceSegments", emptyInt64(), loadReq.SealedSegmentIDs, nil
+	case commonpb.MsgType_HandoffSegments:
+		handoffReq := querypb.HandoffSegmentsRequest{}
+		err = proto.Unmarshal([]byte(t), &handoffReq)
+		if err != nil {
+			return errReturn(taskID, "HandoffSegmentsRequest", err)
+		}
+		pids, sids := c.extractQuerySegmentInfos(taskID, handoffReq.SegmentInfos)
+		log.Info("HandoffSegments", zap.String("detail", fmt.Sprintf("+%v", handoffReq)))
+		return "HandoffSegments", pids, sids, nil
+	default:
+		err = errors.New("inValid msg type when unMarshal task")
+		log.Error("invalid message task", zap.Int("type", int(header.Base.MsgType)), zap.Error(err))
+		return "", emptyInt64(), emptyInt64(), err
+	}
+}
+
+func (c *mck) handleUserEnter(invalidTasks []int64) {
+	if len(invalidTasks) > 0 {
+		fmt.Print("Delete all invalid tasks, [Y/n]:")
+		deleteAll := ""
+		fmt.Scanln(&deleteAll)
+		if deleteAll == "Y" {
+			for _, invalidTask := range invalidTasks {
+				if !c.removeTask(invalidTask) {
+					continue
+				}
+				fmt.Println("Delete task id: ", invalidTask)
+			}
+		} else {
+			deleteTask := ""
+			idMap := int64Map(invalidTasks)
+			fmt.Println("Enter 'Exit' to end")
+			for len(idMap) > 0 {
+				fmt.Print("Enter a invalid task id to delete, [Exit]:")
+				fmt.Scanln(&deleteTask)
+				if deleteTask == "Exit" {
+					return
+				}
+				invalidTask, err := strconv.ParseInt(deleteTask, 10, 64)
+				if err != nil {
+					fmt.Println("Invalid task id.")
+					continue
+				}
+				if _, ok := idMap[invalidTask]; !ok {
+					fmt.Println("This is not a invalid task id.")
+					continue
+				}
+				if !c.removeTask(invalidTask) {
+					continue
+				}
+				delete(idMap, invalidTask)
+				fmt.Println("Delete task id: ", invalidTask)
+			}
+		}
+	}
```

Comment:
print log to all tasks are valid

Reply:
All tasks have printed in the front code.